### DISCUSSION TITLE
Custom timestamp link

### DISCUSSION THREAD

**Discussion by TunaFFish** [lstrzepek/obsidian-yt-transcript#39] [Ideas]
Hi, great plugin, thanks.
I am trying to create a workflow to digest YT video's.
May I suggest a custom timestamp link?

For example I am using YTranscript in tandem with [Obsidian Timestamp Notes](https://github.com/juliang22/ObsidianTimestampNotes) which features timestamp links like this:
~~~
```timestamp 
00:30
````
~~~
For now I could use a regex to replace the YTranscript timestamp link with the Obsidian Timestamp Notes timestamp.
Or better add it on top like this:
~~~
```timestamp 
00:30
````
[00:30](https://www.youtube.com/watch?v=XyZxYzXyZ00&t=30)
~~~

Something to add or replace other timestamp formats would be great because an external link to YT breaks the Obsidian experience (would prevent the "add-hell" on YouTube from taking over)

Cheers!

P.S. here a screenshot:
![Screenshot 2024-01-06 at 21 35 15](https://github.com/lstrzepek/obsidian-yt-transcript/assets/121750989/666da494-ceee-442e-b3fd-462554a79ea0)

**Comment by trevim99** [comment]
very useful
